.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+name: ci
+on: push
+env:
+  CI: true
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+           # pulls all commits (needed for lerna to correctly version)
+          fetch-depth: "0"
+      # pulls all tags (needed for lerna to correctly version)
+      - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - name: Setup Node ${{ matrix.node_version }}
```

Comment:
Where does `matrix.node_version` come from?

Reply:
Can you add comments pointing to docs, like this link? Just makes it easier to click from within the file when needing to.